cmd/dep/init.go

```diff
@@ -43,7 +43,7 @@ func (cmd *initCommand) Register(fs *flag.FlagSet) {}
 type initCommand struct{}
 
 func trimPathPrefix(p1, p2 string) string {
-	if filepath.HasPrefix(p1, p2) {
+	if strings.HasPrefix(p1, p2) {
 		return p1[len(p2):]
 	}
 	return p1
```

Comment:
Wow! That's a much deeper rabbit hole than I expected! 🕳🐇 I'm glad smart people are already looking into this 😃. I've reverted this change and added an `--ignore` flag to tell `staticcheck` to ignore `filepath.HasPrefix` in `cmd/dep/init.go` and `context.go`.